package ex08.example02;

public class TryEx02 {
    public static void main(String[] args) {
        throw new RuntimeException("오류강제발생");
    }
}
